js/data.js

```diff
@@ -22,3 +28,14 @@ window.addEventListener('beforeunload', function(event){
   var dataJSON = JSON.stringify(data);
   localStorage.setItem('data', dataJSON);
 })
+
+var previousNewEntryJSON = localStorage.getItem('newEntry');
+if (previousNewEntryJSON !== null) {
+  newEntry = JSON.parse(previousNewEntryJSON);
+}
+
+window.addEventListener('beforeunload', function (event) {
+  event.preventDefault();
+  var newEntryJSON = JSON.stringify(newEntry);
+  localStorage.setItem('newEntry', newEntryJSON);
+})
```

Comment:
Your `data.js` file should only contain your data model, you can move that rest of the code carefully

Reply:
![image](https://user-images.githubusercontent.com/73401059/101226672-1acc2b80-364a-11eb-94aa-030a330d6808.png)